saleor/payment/migrations/0029_alter_payment_options.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.2 on 2021-05-19 09:50
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
Should we add this permission to staff users?

Reply:
I could do that, but this permission is currently not used by users - no mutations or queries require it. It is only used by apps.